mne/stats/cluster_level.py

```diff
@@ -24,13 +24,15 @@
 from scipy.stats import f as fstat
 from scipy.stats import t as tstat
 
-from .. import BaseEpochs, Evoked, EvokedArray
+from ..epochs import BaseEpochs, EvokedArray
+from ..evoked import Evoked
```

Comment:
Some parts of the MNE API can be imported multiple places. For user scripts, it's OK to do `from mne import Evoked`, but within the MNE package, we should always do `from mne.evoked import Evoked` (or here, ` from ..evoked import Evoked`, which is the equivalent relative import) because it ensures that we avoid any circular import problems. the way to check for this is `pytest mne/tests/test_import_nesting.py`